CMakeLists.txt

```diff
@@ -217,6 +217,9 @@ endif()
 
 if(ENABLE_PIE)
 	add_definitions(-fPIE)
+	if(CMAKE_SYSTEM_NAME MATCHES FreeBSD)
+		add_definitions(-fPIC)
```

Comment:
Shouldn't be needed. PIC does the same thing as PIE, just that it's for shared libraries.
Also we are also passing the linker -pie, so there shouldn't be an issue.